src/io_uring/src/operation/sqe.rs

```diff
@@ -7,17 +7,21 @@ use crate::bindings::io_uring_sqe;
 
 unsafe impl ByteValued for io_uring_sqe {}
 
-pub struct Sqe(pub(crate) io_uring_sqe);
+/// Newtype wrapper over a raw sqe.
```

Comment:
Did you mean `New type` ?

Reply:
Newtype is actually a small Rust design pattern: https://rust-unofficial.github.io/patterns/patterns/behavioural/newtype.html